packages/client/components/SpotlightRoot.tsx

```diff
@@ -20,11 +20,15 @@ const SpotlightRoot = (props: Props) => {
   if (nextReflectionId) {
     reflectionIdRef.current = nextReflectionId
   }
-  const queryRef = useQueryLoaderNow<SpotlightModalQuery>(spotlightModalQuery, {
-    reflectionId: reflectionIdRef.current,
-    searchQuery,
-    meetingId
-  })
+  const queryRef = useQueryLoaderNow<SpotlightModalQuery>(
+    spotlightModalQuery,
+    {
+      reflectionId: reflectionIdRef.current,
+      searchQuery,
+      meetingId
+    },
+    'network-only'
```

Comment:
The inputs to the query are the `spotlightReflectionId` and the `searchQuery`. While these might not change, the results can change significantly if the user or another user groups/ungroups reflections. As the Spotlight is unlikely to be opened frequently, I think it's best if the policy is `network-only` here so that we don't rely on a potentially outdated cache